lib/membrane_h264_ffmpeg/parser.ex

```diff
@@ -21,6 +21,7 @@ defmodule Membrane.H264.FFmpeg.Parser do
   require Membrane.Logger
 
   @required_parameter_nalu [:pps, :sps]
+  @nalu_allowed_before_data [:sei] ++ @required_parameter_nalu
```

Comment:
Shouldn't that be plural? (`@required_parameter_nalus`  `nalus_allowed_before_data`) NALU stands for network abstraction layer unit, so units would be NALUs
Also, both are used as sets of values, so they can be defined as such